### PR TITLE
[GOG]: add support for linux native dlcs in modify install dialog

### PR DESCRIPTION
**Pull request by imLinguin**
also automatically run post install script after completed download

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Review by flavioislima — APPROVED** [review]
LGTM

File: src/backend/storeManagers/gog/games.ts [hunk 1]
```diff
@@ -1140,6 +1152,17 @@ export async function update(
     (isWindows || existsSync(gameSettings.winePrefix))
   ) {
     await setup(appName, gameObject, false)
+  } else if (gameObject.platform === 'linux') {
+    const installer = join(gameObject.install_path, 'support/postinst.sh')
+    if (existsSync(installer)) {
+      logInfo(`Running ${installer}`, LogPrefix.Gog)
+      await spawnAsync(installer, []).catch((err) =>
```
Comment (CommandMC) [inline src/backend/storeManagers/gog/games.ts:1159]: Are we sure these scripts always have their executable bit set?

**Review by CommandMC — APPROVED** [review]
I don't think I have any games I could test this with, but code looks good

File: src/backend/storeManagers/gog/games.ts (hunk 1 above)
Comment (imLinguin) [reply src/backend/storeManagers/gog/games.ts:1159]: pretty sure, yeah, they are automatically added to the zip and include the flag. I guess we could add a safeguard to always set it.

Comment (CommandMC) [reply src/backend/storeManagers/gog/games.ts:1159]: I see. Let's add that if we run into any issues, no point in adding safeguards where they aren't necessary

**Pull request merged**